### PR TITLE
Slightly updated officers page

### PR DESCRIPTION
**Pull request by chriskuehl**

### CODE REVIEW

File: ocfweb/docs/views/officers.py [hunk 1]
```diff
@@ -106,19 +103,21 @@ def officer_from_uid(uid):
...
+    Term('Summer 2015', gms=['nickimp'], sms=['ckuehl']),
     Term('Fall 2015', gms=['nickimp'], sms=['ckuehl']),
-]))
+]
 
 
 def officers(doc, request):
-    return render_to_response(
+    return render(
+        request,
```
Comment (chriskuehl) [inline ocfweb/docs/views/officers.py]: render_to_response was deprecated

File: ocfweb/docs/templates/officers.html [hunk 2]
```diff
@@ -29,26 +23,24 @@
           href="/about/staff">weekly meetings</a>.
         </p>
       <h3>
-        General managers
-        &lt;<a href="mailto:gm@ocf.berkeley.edu">gm@ocf.berkeley.edu</a>&gt;
```
Comment (chriskuehl) [inline ocfweb/docs/templates/officers.html]: I hate these emails. One would email gm@ocf thinking they are emailing only the GM, but unwittingly also email 10 other officers.

File: ocfweb/docs/views/officers.py [hunk 3]
```diff
@@ -106,19 +103,21 @@ def officer_from_uid(uid):
     Term('Summer 2013', gms=['daradib'], sms=['tzhu']),
     Term('Fall 2013', gms=['daradib'], sms=['tzhu']),
     Term('Spring 2014', gms=['nickimp'], sms=['ckuehl']),
+    Term('Summer 2014', gms=['nickimp'], sms=['ckuehl']),
     Term('Fall 2014', gms=['nickimp'], sms=['ckuehl']),
     Term('Spring 2015', gms=['nickimp'], sms=['ckuehl']),
+    Term('Summer 2015', gms=['nickimp'], sms=['ckuehl']),
```
Comment (matthew-mcallister) [inline ocfweb/docs/views/officers.py]: We've never done elections for the summer, so I don't think this adds much except inflating the number of times `nickimp` and `ckuehl` appear on this page (=P)

**Comment by matthew-mcallister** [comment]
Changes look good.

File: ocfweb/docs/views/officers.py (hunk 3 above)
Comment (chriskuehl) [inline ocfweb/docs/views/officers.py]: heh, you caught me

just continuing the trend though. we did summer 2011, 2012, and 2013.

Comment (chriskuehl) [inline ocfweb/docs/views/officers.py]: (i actually don't care though, if you think it's silly to list summer I'm fine with removing them)

Comment (matthew-mcallister) [inline ocfweb/docs/views/officers.py]: Go ahead and merge, we'll just continue to list summers in the future for consistency.

**Pull request merged**